grunnbeløp.json

```diff
@@ -1,5 +1,13 @@
 {
   "grunnbeløp": [
+    {
+      "dato": "2023-05-01",
+      "grunnbeløp": 118620,
+      "grunnbeløpPerMåned": 9885,
+      "gjennomsnittPerÅr": 116372,
+      "omregningsfaktor": 1.0641,
```

Comment:
Mulig det burde brukes 6 desimaler her for at det skal bli helt korrekt, for det blir et avvik på 2-3 kr med avrundet omregningsfaktor: 
111477*1.064076 = 118620,000252
111477*1.0641 = 118622,6757